docs/source/using/experimental-features.adoc

```diff
@@ -151,3 +151,51 @@ To get the status of the DNS server:
 ----
 $ minishift dns status
 ----
+
+[[run-against-an-existing-machine]]
+== Run against an Existing Machine
+
+{project} can be run against an existing machine using `vm-driver` as `generic` along with the required options.
+To use an existing machine with {Project} , it needs to be configured as follows:
+
+[NOTE]
+====
+CentOS7/RHEL7/Fedora(>26) are the suggested Linux distributions for this feature.
+====
+
+Lets assume that 10.1.1.1 is the IP of machine which you want use to run against {project}.
+
+Establish password less SSH from the host to existing machine:
+
+----
+Host$ ssh-copy-id <user>@10.1.1.1  // Make sure user have sudo access without password or use root
+Host$ ssh <user>@10.1.1.1  // Make sure this login without password
+----
+
+Prerequisite steps for configuring the existing machine:
+
+----
+Remote_VM$ yum install -y docker net-tools
+Remote_VM$ systemctl start docker
+Remote_VM$ systemctl enable docker
+----
+
+You also need to allow `2376`, `8443` port on the machine from firewall to have communication from the host.
+
+----
+firewall-cmd --permanent --add-port 2376/tcp
+firewall-cmd --permanent --add-port 8443/tcp
+firewall-cmd --reload
+----
+
+Run {project} against remote Machine as follows:
+
+----
+Host$ export MINISHIFT_ENABLE_EXPERIMENTAL=y
+Host$ minishift start --vm-driver generic --remote-ipaddress 10.1.1.1 --remote-ssh-user <username> --remote-ssh-key <private_ssh_key>
+----
+
+[NOTE]
+====
+`--remote-ssh-key` flag value should be location of the private ssh key on the host.
```

Comment:
"The value of the `--remote-ssh-key` flag must be the location of a private SSH key on the host machine."